### PR TITLE
Edits to the documentation from COSMO Retreat

### PR DESCRIPTION
**Pull request by rosecers**
- [x] more fleshed out landing page

### CODE REVIEW

File: docs/source/intro.rst [hunk 1]
```diff
@@ -12,4 +12,12 @@ Currently, scikit-COSMO contains models described in [Imbalzano2018]_, [Helfrech
 as some modifications to sklearn functionalities and minimal datasets that are useful within the field
 of computational materials science and chemistry.
 
+- FINGERPRINT SELECTION:
+   Multiple data sub-selection modules, for selecting the most relevant features and samples out of a large set of candidates.
+      * CUR decomposition: an iterative feature selection method that has been developed to deal with data where the information provided by the singular vectors cannot be properly interpreted :ref:`CUR-api`.
```
Comment (rosecers) [inline docs/source/intro.rst:17]:
```suggestion
      * CUR decomposition: an iterative feature selection method based upon the singular value decoposition.
```

File: docs/source/intro.rst [hunk 2]
```diff
@@ -12,4 +12,12 @@ Currently, scikit-COSMO contains models described in [Imbalzano2018]_, [Helfrech
...
+      * PCov-CUR decomposition extends upon CUR by using augmented right or left singular vectors inspired by Principal Covariates Regression :ref:`PCov-CUR-api`.
+      * Farthest Point-Sampling (FPS): a common selection technique intended to exploit the diversity of the input space. The selection of the first point is made at random or by a separate metric :ref:`FPS-api`
+      * PCov-FPS extends upon FPS much like PCov-CUR does to CUR :ref:`PCov-FPS-api`.
+      * Voronoi FPS: construct a Voronoi tessellation obtained using the Hausdorff distance and with centers the FPS selected points :ref:`Voronoi-FPS-api`.
```
Comment (rosecers) [inline docs/source/intro.rst:21]:
```suggestion
      * Voronoi FPS: conduct FPS selection, taking advantage of Voronoi tessellations to accelerate selection :ref:`Voronoi-FPS-api`.   
```

File: docs/source/intro.rst [hunk 3]
```diff
@@ -12,4 +12,22 @@ Currently, scikit-COSMO contains models described in [Imbalzano2018]_, [Helfrech
...
+      * :ref:`_CUR-api:` decomposition: an iterative feature selection method based upon the singular value decoposition.
+      * :ref:`PCov-CUR-api` decomposition extends upon CUR by using augmented right or left singular vectors inspired by Principal Covariates Regression.
+      * :ref:`FPS-api`: a common selection technique intended to exploit the diversity of the input space. The selection of the first point is made at random or by a separate metric.
+      * :ref:`PCov-FPS-api` extends upon FPS much like PCov-CUR does to CUR.
+      * :ref:`Voronoi-FPS-api`: conduct FPS selection, taking advantage of Voronoi tessellations to accelerate selection.   
+
+- Reconstruction Measures:
+   A set of modules to compare different sets of descriptors (or featurizations) `F` and `F'`, and different ways of transforming them by means of metrics and kernels, in terms of the structure of the feature space that they induce [Goscinski2021]_ .
+      * :ref:`GRE-api` (GRE) a simple, easily-interpretable measure of the relative expressive power of $F$ and $F'$. Defined as the root mean-square error that one incurs when using the feature matrix $X_F$ to linearly regress $X_F'$.
```
Comment (agoscinski) [inline docs/source/intro.rst:25]: TODO for myself: rework text here a little bit

**Comment by agoscinski** [comment]
Here a preview of the new doc

new_formatting

I dont know why, locally I get formatting issues for the 2-cv ridge class. I would nevertheless merge this and check afterwards if these persists in the online version.

weird_format

I had to add a version into requirement for Jinja2 because of the issue mentioned in the commit.

**Review by agoscinski — APPROVED** [review]

**Pull request merged**